.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Was this added by mistake? I'm not against devcontainers, but I haven't used them much, which means that I'll reach out every time this needs maintenace/update/...

Reply:
Any reason why you're not using R 4.4.x here?

Reply:
No particular reason, I pretty much copies over the config from one of my ongoing applications :D 

Sorry to introduce this burden, but I literally could not test the app locally dues to some problem with `libpq` - even tho I have pg v17 installed. So I thought that it might be useful for some other contributors.

Reply:
Pushed a change to use the release version of R.